<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemeFrame</class>
 <widget class="QWidget" name="SchemeFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="RegularRadio">
     <property name="text">
      <string>Full backups</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="IncrementalRadio">
     <property name="text">
      <string>Incremental backups</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Number of increments</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="LengthChain">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="specialValueText">
      <string>No limit</string>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Use as reference storage</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="ReferenceStorageComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of stored full backups</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="NumStored">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="specialValueText">
      <string>No limit</string>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Temporary directory</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="TemporaryDirEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="BrowseTemporaryDirButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="DefaultTemporaryPathAction">
   <property name="text">
    <string>Use default</string>
   </property>
   <property name="toolTip">
    <string>Use default</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>IncrementalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IncrementalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>LengthChain</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IncrementalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IncrementalRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>ReferenceStorageComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
